src/main/resources/outcomeCodeLookup.txt

```diff
@@ -348,5 +351,4 @@
 32-03-01|ADDRESS_NOT_VALID
 32-03-02|ADDRESS_NOT_VALID
 32-03-03|ADDRESS_NOT_VALID
-32-03-04|ADDRESS_NOT_VALID
-32-03-05|ADDRESS_NOT_VALID
```

Comment:
did you mean to remove this?

Reply:
I didn't remove any. I only added new outcome codes.